samples/sql-persistence/simple/sample_prereqs_sqlpersistence_[2,).partial.md

```diff
@@ -0,0 +1,23 @@
+
+### MS SQL Server
```

Comment:
We need different prereqs for different versions: MySQL and SQL Server only for 1.0 and add Oracle for 2.0. I don't know how to mix includes and partial here. I thought this was impossible for a partial to use an include.

Is there a way to do this?